src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -28,6 +28,7 @@
   <ItemGroup>
 	<ProjectReference Include="../Microsoft.SqlTools.Hosting/Microsoft.SqlTools.Hosting.csproj" />
 	<ProjectReference Include="../Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj" />
+	<ProjectReference Include="..\Microsoft.SqlTools.Serialization\Microsoft.SqlTools.Serialization.csproj" />
```

Comment:
The includes above use `/` and this one uses `\`. Does it make a difference? Good to be consistent in any case

Reply:
Don't see this addressed yet